changelogs/fragments/481-ec2_instance-wait_sanity.yml

```diff
@@ -0,0 +1,11 @@
+breaking_changes:
+- >-
+  ec2_instance - if plays require the old behavior of ``ec2_instance``, waiting for
```

Comment:
These reads a little abruptly to me.  Maybe something like:
Instance wait for state behaviour has changed. If plays require the old ...